bin/yaml/snowball.yaml

```diff
@@ -0,0 +1,13 @@
+compilers:
+  snowball:
+    type: tarballs
+    compression: gz
+    dir: snowball-{name}
+    create_untar_dir: true
+    strip_components: 1
+    build: snowball-{name}-release
+    toolchain: cmake
+    check_exe: bin/snowball --version
+    targets:
+      - name: '0.0.7'
+        url: https://github.com/snowball-lang/snowball/releases/download/v0.0.7-hotfix/snowball-ce-specific-x86_64.tar.gz
```

Comment:
Ideally the URL should be related to the `{name}` but as this has `hotfix` in the middle I guess that's just a special case for this release?

Reply:
Yeah, this is a one time case because version 0.0.7 did not include Ce-specific bi aries.